docs/usage/notes.rst

```diff
@@ -208,6 +208,9 @@ To activate append-only mode, set ``append_only`` to 1 in the repository config:
 
     borg config /path/to/repo append_only 1
 
+Note that you can go back-and-forth between normal and append-only operation by editing
+the configuration file, it's not a "one way trip".
```

Comment:
maybe don't say "by editing the configuration file" (that sounds like `vim config`) just below showing the `borg config` command that does that.

Reply:
That sentence was already in the docs, I just moved it to a more logical place. I could change it to say "with `borg config`"?

Reply:
ok, please do.